server/data/reportLogClient.ts

```diff
@@ -0,0 +1,31 @@
+import { QueryPerformer } from './dataAccess/db'
+
+type Action =
```

Comment:
enum?

Reply:
it's actually easier to consume as a string union - you don't need to actually reference the enum type in the client and you still get compile time safety and code completion